### ISSUE TITLE
AI menu doesn't show up (GIMP 2.10)

### ISSUE THREAD

**Issue by junosha** [blueturtleai/gimp-stable-diffusion#8]
Copied gimp-stable-diffusion.py into plug-ins folder, permissions changed, but AI menu does not show up. How can I troubleshoot this?

**Comment by blueturtleai** [comment]
Afaik there is no log file which you could check for errors. Which OS do you use?

**Comment by junosha** [comment]
macOS 12.5.1

**Comment by blueturtleai** [comment]
Please check, if you can run "Filters > Render > Spyrogimp". This plugin is also Python based.

**Comment by blueturtleai** [comment]
If this works, please check everything mentioned in the answer on stackoverflow. Also set the file permissions as mentioned in the reply of the author at the very bottom. https://stackoverflow.com/questions/60824691/gimp-python-script-not-showing-in-menu

**Comment by junosha** [comment]
Thanks. Checked the permissions again, they were all set correctly. Tried to run the file in VS Code and got this:

Traceback (most recent call last):
  File "/Users/admin/Library/Application Support/GIMP/2.10/plug-ins/gimp-stable-diffusion.py", line 5, in <module>
    import urllib2
ModuleNotFoundError: No module named 'urllib2'

I installed Python 2.7.18 and tried again, but still the AI menu didn't show. Strange
